src/index.html

```diff
@@ -7,6 +7,342 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+
+      <header class="header">
+        <a href="#" class="logo">
+          <img
+             src="./images/logo.png"
+             alt="Moyo logo"
+             class="logo__image"
+           >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link"
+               data-qa="nav-hover"> Laptops & computers </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+        <div class="cards">
+          <div class="card" data-qa="card">
+            <img src="./images/imac.jpeg" alt="iMac"
+                class="card__img">
+
+            <div class="card__discription">
+              APPLE A1419 iMac 27" Retina
+              <br>
+              5K Monoblock (MNED2UA/A)
+            </div>
+
+            <div class="card__product-code">Product code: 195434</div>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__rating-reviews">
+                  Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-price">Price:</p>
+              <p class="card__price-num">69 999 грн</p>
+            </div>
+
+            <button class="card__buy" href="#"
+             data-qa="card-hover">
```

Comment:
```suggestion
            <button 
              class="card__buy" 
              href="#"
              data-qa="card-hover"
           >
```
Consider fixing indents according to the code style guide